egs2/TEMPLATE/asr1/db.sh

```diff
@@ -59,6 +59,7 @@ LABOROTV=
 TEDXJP=
 LIBRISPEECH=downloads
 LIBRILIGHT_LIMITED=
+LIBRILIGHT="downloads"
```

Comment:
"downloads" means the dataset can be automatically downloaded by the data.sh script. Do we support this auto-downloading for librilight?